frontend/src/app/components/MainInstallButton.js

```diff
@@ -20,27 +20,14 @@ export default class MainInstallButton extends React.Component {
   }
 
   render() {
-    const { isFirefox, hasAddon } = this.props;
+    const { isFirefox, isMinFirefox, hasAddon } = this.props;
     const isInstalling = this.state.isInstalling && !hasAddon;
     const experimentTitle = ('experimentTitle' in this.props &&
                              this.props.experimentTitle);
 
     return (
       <div>
-        {(!isFirefox) ? (
-          <div>
-              <span data-l10n-id="landingDownloadFirefoxDesc" className="parens">(Test Pilot is available for Firefox on Windows, OS X and Linux)</span>
-              <a href="https://www.mozilla.org/firefox" className="button primary download-firefox">
-                <div className="button-icon">
-                  <div className="button-icon-badge"></div>
-                </div>
-                <div className="button-copy">
-                  <div data-l10n-id="landingDownloadFirefoxTitle" className="button-title">Firefox</div>
-                  <div data-l10n-id="landingDownloadFirefoxSubTitle" className="button-description">Free Download</div>
-                </div>
-              </a>
-          </div>
-        ) : (
+        {(isMinFirefox) ? (
```

Comment:
If we're going to use a ternary, can you put both chunks in methods? The end is hard to find.

`(isMinFirefox ? this.renderInstallButton : renderAltButton)()`